docs/developers/README.md

```diff
@@ -727,3 +696,19 @@ from the message is internal to Tekton Pipelines, used for book-keeping, and
 some is distributed across a number of fields of the `TaskRun's` `status`. For
 example, a `TaskRun's` `status.taskResults` is populated from the termination
 message.
+
+## Experimentally Disabling PipelineRef and TaskRef Resolution
```

Comment:
This chunk is a rebase artifact I think, should be removed to match latest main.

Reply:
Done.